pkg/scheduler/apis/config/types.go

```diff
@@ -55,6 +55,9 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta
 
+	// CPU threads
```

Comment:
"Parallelism defines the amount of parallelism that should be used in scheduler algorithms and some plugins."